docs/installation.md

```diff
@@ -25,6 +25,16 @@ The Lychee gallery has a few system requirements. You will need to make sure you
    		- Tokenizer
    		- XML
    		- ZIP
+   	- These PHP extensions are necessary if you are running a FreeBSD system:
+   	 	- Simplexml
+		- Dom
+   	 	- Session
+   	 	- Zlib
+   	- For database, you will need these PHP extensions:
+   	 	- SQLite3 (For Lychee's inbuilt SQLite3)
+   	 	- MySQLi (For MySQL or MariaDB)
+   	 	- PDO_MySQL (For MySQL or MariaDB)
+- You also need install composer to complete installation, visit https://getcomposer.org/download/ for help.
```

Comment:
I'm not sure whether this is needed in this section, but if it's to be included here it should be more specific - installing without composer is possible (from a release).

```suggestion
- To install from git you will also need composer. See [below](#from-the-master-branch) for details.
```

Reply:
Yeah, I install from git because it failed from release at first.